static/webpack-config.js

```diff
@@ -25,11 +26,29 @@ module.exports = function () {
     });
   }
 
+  const globalConfigPath = `./${jamboConfig.dirs.config}/global_config.json`;
+  let globalConfig = null;
+  if (fs.existsSync(globalConfigPath)) {
+    globalConfigRaw = fs.readFileSync(globalConfigPath, 'utf-8');
+    globalConfig = parse(globalConfigRaw);
+  }
+
+  const useJWT = globalConfig && globalConfig.useJWT
+  let jamboInjectedData = process.env.JAMBO_INJECTED_DATA || null;
+  jamboInjectedData = jamboInjectedData && JSON.parse(jamboInjectedData);
+
+  const getSecuredJamboInjectedData =
+    require(`./${jamboConfig.dirs.output}/static/webpack/getSecuredJamboInjectedData.js`);
+
+  const updatedJamboInjectedData = useJWT 
+    ? getSecuredJamboInjectedData(jamboInjectedData)
+    : jamboInjectedData
+
   const plugins = [
     new MiniCssExtractPlugin({ filename: '[name].css' }),
     ...htmlPlugins,
-    new webpack.EnvironmentPlugin({
-      JAMBO_INJECTED_DATA: null
+    new webpack.DefinePlugin({
+      'process.env.JAMBO_INJECTED_DATA': JSON.stringify(JSON.stringify(updatedJamboInjectedData))
```

Comment:
I prefer to leave it stringified twice so that we don't have to update outside classes, but let me know what you think! I tried the single stringified approach and it seems to cause some problems